### PR TITLE
Add-image-source

### PR DESCRIPTION
**Pull request by jaresty**
- apparently the gpt-4o model supports images out of the box

https://platform.openai.com/docs/guides/vision

### CODE REVIEW

**Comment by jaresty** [comment]
I considered splitting up the prompts since many prompts don't apply to text and images both, however sometimes you take screenshots of things that include text and images. I think that rather than trying to artificially split them it may be perfectly reasonable to leave them altogether and rely on users to make a distinction about which prompt they want to use.

**Comment by jaresty** [comment]
This might be a better approach: https://github.com/C-Loftus/talon-ai-tools/pull/94

File: GPT/gpt.talon [hunk 1]
```diff
@@ -20,6 +20,11 @@ model snip <user.modelPrompt> [{user.modelSource}] [{user.modelDestination}]:
     result = user.gpt_apply_prompt(modelPrompt, text, "snip")
     user.gpt_insert_response(result, modelDestination or "", "snip")
 
+# Applies a prompt to images in the clipboard
+model <user.modelPrompt> image [{user.modelDestination}]:
```
Comment (C-Loftus) [inline GPT/gpt.talon:24]: I like the syntax :)

**Comment by C-Loftus** [comment]
Started to review this but this is superceded by the  newest open PR right?

**Comment by jaresty** [comment]
Yes - I think the newest PR is the best of the options (IMO).  I left this here as an alternative idea.

**Pull request closed without merging**